build/teamcity-acceptance.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 set -euxo pipefail
+
+export COCKROACH_SKIP_FLAKY_TESTS=true
```

Comment:
Unfortunately, this won't work - `builder.sh` only passes a particular set of environment variables through to the Docker environment. You'll need to add `COCKROACH_SKIP_FLAKY_TESTS` to the list, or just change `builder.sh` to pass through all of `env`.